tests/integration_tests/test_mlflow.py

```diff
@@ -0,0 +1,109 @@
+from mlflow.tracking import MlflowClient
+import py
+
+import optuna
+from optuna.integration.mlflow import MLflowCallback
+
+
+def _objective_func(trial, x_max=1.0):
```

Comment:
A nitpick, if you have time, we could remove and hard code the `x_max` argument. It's not passed any non-default value anyway.

Reply:
changed that